### PR TITLE
fix(ui5-side-navigation): fixed accessibility of the Popover

### PR DESCRIPTION
**Pull request by LidiyaGeorgieva**
PartOf: #5827

### CODE REVIEW

File: packages/fiori/src/SideNavigationItem.js [hunk 1]
```diff
@@ -80,6 +80,18 @@ const metadata = {
 		 title: {
 			type: String,
 		},
+
+		/**
+		 * Defines if the item should be collapsable or not.
+		 * It is true, for example, for the items inside the Popup of the Side Navigation
```
Comment (BorisDafov) [inline packages/fiori/src/SideNavigationItem.js:86]: maybe:
Popover of the Side Navigation

File: packages/main/src/TreeItem.js [hunk 2]
```diff
@@ -182,6 +182,18 @@ const metadata = {
 			noAttribute: true,
 		},
 
+		/**
+		 * Defines if the item should be collapsable or not.
+		 * It is true, for example, for the items inside the Popup of the Side Navigation
```
Comment (BorisDafov) [inline packages/main/src/TreeItem.js:187]: maybe:
Popover of the Side Navigation

File: packages/main/src/TreeListItem.js [hunk 3]
```diff
@@ -170,6 +170,18 @@ const metadata = {
 			noAttribute: true,
 		},
 
+		/**
+		 * Defines if the item should be collapsable or not.
+		 * It is true, for example, for the items inside the Popup of the Side Navigation
```
Comment (BorisDafov) [inline packages/main/src/TreeListItem.js:175]: maybe:
Popover of the Side Navigation

File: packages/main/src/TreeListItem.js [hunk 4]
```diff
@@ -170,6 +170,18 @@ const metadata = {
 			noAttribute: true,
 		},
 
+		/**
+		 * Defines if the item should be collapsable or not.
```
Comment (BorisDafov) [inline packages/main/src/TreeListItem.js:174]: collapsible

**Comment by BorisDafov** [comment]
From UA perspective, I approve the pull request...

**Review by TeodorTaushanov — APPROVED** [review]

File: packages/main/src/TreeListItem.js [hunk 5]
```diff
@@ -170,6 +170,18 @@ const metadata = {
 			noAttribute: true,
 		},
 
+		/**
+		 * Defines if the item should be collapsible or not.
```
Comment (dobrinyonkov) [inline packages/main/src/TreeListItem.js:174]: As I see this property is not used when creating the `ui5-li-tree` instances in the `Tree.hbs`, we can remove the definition here. Using it in the `get requiresToggleButton()` is enough to hide the toggle button.

**Review by TeodorTaushanov — APPROVED** [review]

**Pull request merged**